pkg/gardenlet/apis/config/v1alpha1/types.go

```diff
@@ -490,6 +490,33 @@ type SNIIngress struct {
 
 // ETCDConfig contains ETCD related configs
 type ETCDConfig struct {
+	// ETCDController contains config specific to ETCD controller
+	ETCDController *ETCDController `json:"etcdController,omitempty"`
+	// CustodianController contains config specific to custodian controller
+	CustodianController *CustodianController `json:"custodianController,omitempty"`
+	// BackupCompactionController contains config specific to backup compaction controller
+	BackupCompactionController *BackupCompactionController `json:"backupCompactionController,omitempty"`
+}
+
+// ETCDController contains config specific to ETCD controller
+type ETCDController struct {
+	// Workers specify number of worker threads in ETCD controller
+	// Defaults to 3
+	Workers *int64 `json:"etcdControllerWorkers,omitempty"`
+}
+
+// CustodianController contains config specific to custodian controller
+type CustodianController struct {
+	// Workers specify number of worker threads in custodian controller
+	// Defaults to 3
```

Comment:
```suggestion
	// Defaults to 3
	// +optional
```

Reply:
Shouldn't this be `10` instead? Since it defaults to 10 [here](https://github.com/abdasgupta/gardener/blob/6367c9859c45414a50c5e5682e09d0a24a9dfadb/pkg/operation/botanist/component/etcd/bootstrap.go#L331)